contract.md

```diff
@@ -4,6 +4,8 @@
 
 _At a high level, what do you most want to get out of this project, and why?_
 
+_I want to figure out how to write a language that could help me perform task that I routinely do. I want this language to have use for me outside of this class._
```

Comment:
I had a similar motivation for my project. The regularly useful results seem like a good motivation for putting work into the project.